src/features/hotspots/setup/HotspotSetupDiagnosticsScreen.tsx

```diff
@@ -108,12 +108,19 @@ const HotspotSetupDiagnosticsScreen = () => {
   const handleSendReport = useCallback(() => {
     const supportEmail = getMakerSupportEmail(onboardingRecord?.maker?.id)
     const descriptionInfo = t('hotspot_settings.diagnostics.desc_info')
+
+    if (!supportEmail) {
+      const maker = onboardingRecord?.maker?.name || 'your Hotspot manufacturer'
```

Comment:
Could you translate these strings in the alert?

Reply:
Added